### PR TITLE
Edwardkoval patch 1

### PR DESCRIPTION
**Pull request by edwardkoval**

### CODE REVIEW

File: Samples/PowerShell/Readme.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+## Questions and comments
+
+We'd love to get your feedback about the Secuirty API PowerShell sample. You can send your questions and suggestions to us in the Issues section of this repository.
+
+Your feedback is important to us. Connect with us on Stack Overflow. Tag your questions with [microsoft-graph-security].
+
+
+## Additional resources
+* [Microsoft Graph API documentation](https://developer.microsoft.com/en-us/graph/docs)
```
Comment (preetikr) [inline Samples/PowerShell/Readme.md]: Add link to Microsoft Graph security Learn overview documentation and call it as 'Microsoft Graph Security Documentation'

File: Samples/PowerShell/Readme.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+* [Microsoft Graph Portal](https://developer.microsoft.com/en-us/graph/graph-explorer)
+* [Microsoft code samples](https://developer.microsoft.com/en-us/graph/code-samples-and-sdks)
+* [Graph Security Documentation](https://developer.microsoft.com/en-us/graph/docs/concepts/security-concept-overview)
```
Comment (preetikr) [inline Samples/PowerShell/Readme.md]: This is the link we need - we can remove the Graph documentation reference - you can add reference to the V1.0 Schema that will enable developers to look and integrate better.

File: Samples/PowerShell/Readme.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* These samples were adopted to call the Security API, visit [powershell-intune-samples](https://github.com/microsoftgraph/powershell-intune-samples) for the original repo.
```
Comment (preetikr) [inline Samples/PowerShell/Readme.md]: Rephrase this as 'The authentication to Microsoft Graph is leveraged from the powershell-intune samples' (you can link to it). 
Remove the 'please visit etc. details.

**Comment by mshalev** [comment]
This went out to all subscribed – better in future to little ‘r, I think 😊
(sent from my public email)

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
